collect_app/src/main/res/values/strings.xml

```diff
@@ -701,4 +701,5 @@
     <string name="copying_media_files_failed">Copying media files failed</string>
     <string name="qr_code_not_found">QR Code not found in the selected image</string>
     <string name="not_granted_permission">The activity you are trying to start requires a permission which is not granted.</string>
+    <string name="file_not_created_via_gs">The submission url refers to a spreadsheet uploaded to Google Drive, not created via Google Sheets.</string>
```

Comment:
I think this is uncommon enough that it would be fine to leave as a literal string and not have it translated.

Reply:
You are right. Fixed.